services/hooks/package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "heroku-prebuild": "echo $SOURCE_VERSION > .git-version",
     "lint": "eslint src/*.js test/*.js",
-    "test": "mocha test/*_test.js"
```

Comment:
Was this temporary?  You can achieve the same thing by running `node_modules/.bin/mocha test/expires_test.js`